<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>761</width>
      <height>381</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="pag1">
     <attribute name="title">
      <string>Tab 1</string>
     </attribute>
     <layout class="QGridLayout" name="gridLayout">
      <item row="6" column="2">
       <widget class="QLineEdit" name="inTxt"/>
      </item>
      <item row="4" column="2">
       <widget class="QLineEdit" name="freqTxt"/>
      </item>
      <item row="3" column="2">
       <widget class="QLabel" name="freqlbl">
        <property name="text">
         <string>Frequencia</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="tensaolbl">
        <property name="text">
         <string>Tensão</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="potlbl">
        <property name="text">
         <string>Potência</string>
        </property>
       </widget>
      </item>
      <item row="5" column="2">
       <widget class="QLabel" name="inlbl">
        <property name="text">
         <string>Corrente Nominal (In)</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="tensaoTxt"/>
      </item>
      <item row="1" column="2">
       <widget class="QLineEdit" name="potTxt"/>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="fslbl">
        <property name="text">
         <string>Fator de Serviço (FS)</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="fstxt"/>
      </item>
      <item row="5" column="1">
       <widget class="QLabel" name="ipinlbl">
        <property name="text">
         <string>Ip/In</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QLineEdit" name="ipinTxt"/>
      </item>
      <item row="13" column="1" rowspan="2" colspan="3">
       <widget class="QPushButton" name="calcBtn">
        <property name="text">
         <string>DIMENSIONAR!</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="pag2">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
